R/dependencies.R

```diff
@@ -124,7 +124,7 @@ dashNoUpdate <- function() {
 #' @rdname selectors
 #' @export
 #' @examples
-#' if (interactive() && require(dash)) {
+#' if (interactive() ) {
 #'   library(dashCoreComponents)
 #'   library(dashHtmlComponents)
```

Comment:
🔪  (I made a similar comment above, are there more cases like this?)

Reply:
Yes, for example `man/dash.Rd` has a bunch

Reply:
I've updated it in d66d720 for this branch. However, for future dcc/html/table updates, we'll also need to make PR's in those repos once this PR is merged to update the `dash-info.yaml` and scrub the library imports and `require(dash)` from the examples. It should be a relatively minor update and I can open those PR's after this one is merged.